x-pack/plugin/esql/qa/testFixtures/src/main/resources/eval.csv-spec

```diff
@@ -215,3 +215,17 @@ emp_no:integer  | foldable:keyword | folded_mv:keyword
 10001           | "foo,bar"        | [foo, bar]
 10002           | "foo,bar"        | [foo, bar]
 ;
+
+randomNumberNoSeed
+row z=1 | eval a=random() | eval b=random() | eval r=case(a!=b, 1, 0) | where r==1 | eval r=to_int(case(a<1, 1, 0)) | keep r;
```

Comment:
This test will eventually fail randomly (I think), but it's so unlikely that it's absolutely fine for me to keep it as it is